services/petstore/cmd/controller/main.go

```diff
@@ -0,0 +1,94 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+	"os"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrlrt "sigs.k8s.io/controller-runtime"
+
+	ackrt "github.com/aws/aws-service-operator-k8s/pkg/runtime"
+
```

Comment:
nit: empty line. can be grouped together with other same package imports

Reply:
so here I separate out imports of the ACK runtime and service-controller-specific imports (below)

Reply:
ah, yeah, the ackrt import should be in the second group. :)

Reply:
Thanks! Updating in next pull request since this pull request is merged now.